src/nft/components/layout/Radio.tsx

```diff
@@ -0,0 +1,22 @@
+import clsx from 'clsx'
+import { Box } from 'nft/components/Box'
+import * as styles from 'nft/components/layout/Radio.css'
+import { MouseEvent } from 'react'
+
+interface RadioProps {
+  hovered: boolean
+  checked: boolean
+  onClick: (e: MouseEvent) => void
+}
+
+export const Radio = ({ hovered, checked, onClick }: RadioProps) => {
+  return (
+    <Box
+      as="label"
+      className={clsx(checked ? styles.selectedRadio : hovered ? styles.blueBorderRadio : styles.greyBorderRadio)}
```

Comment:
if you're gonna use clsx for this i'd recommend using the object form. otherwise you can just remove it and leave the ternaries

Reply:
yep sorry forgot to remove